src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/custom.py

```diff
@@ -195,6 +195,51 @@ def update_adla_catalog_credential(client,
     client.update_credential(account_name, database_name, credential_name, update_params)
 
 
+# customizations for listing catalog items that support multiple ancestor levels of listing.
+def list_catalog_tables(client,
+                        account_name,
+                        database_name,
+                        schema_name=None):
+    if not schema_name:
+        return client.list_tables_by_database(account_name, database_name)
+
+    return client.list_tables(account_name, database_name, schema_name)
+
+def list_catalog_views(client,
```

Comment:
At this point, the CI build hasn't finished yet, but I guess the code style check will fail. Please run `check_style --module dla --pep8 --pylint` to detect all code styles issue.